alf/layers.py

```diff
@@ -100,8 +109,41 @@ def unflatten(self, tensor):
                      tuple(tensor.shape[1:])))
 
 
+class Module(nn.Module):
+    """Alf's nn.Module wrapper"""
+
+    def __init__(self):
+        super().__init__()
+        self.rollingout = False
+        self.summary_name = ""
```

Comment:
self._summary_name
and use commented property accessor/setter for it since it's publicly accessible